### PR TITLE
fix(logs) logs should not color words with the ERR substring

### PR DESCRIPTION
**Pull request by corykitchens**
Addresses #1312 

Removes the element `ERR` from the `fatalCodes` slice

This ensures that any Event struct containing the substring "ERR" will no longer be color-coded. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by iamhopaul123 — CHANGES_REQUESTED** [review]
Thanks for contributing! I believe this can fix the issue for now but in the long run we might still have similar problems if there's any keyword containing "fatal" or something else that will be colored unexpectedly. Would you mind taking a different route like preventing any keyword with substring `fatalCodes` to be colored?

**Review by iamhopaul123 — APPROVED** [review]
This is awesome! Thank you so much for contributing!

File: internal/pkg/aws/cloudwatchlogs/event.go [hunk 1]
```diff
@@ -50,3 +51,17 @@ func (l *Event) shortLogStreamName() string {
...
+
+// colorCodeMessage returns the given message with color applied to every occurence of code
+func colorCodeMessage(message string, code string, givenColor *c.Color) string {
+	if c.NoColor {
+		return message
+	}
+	words := strings.Split(message, " ")
+	for i, word := range words {
+		if word == code {
+			words[i] = givenColor.Sprint(word)
```
Comment (iamhopaul123) [inline internal/pkg/aws/cloudwatchlogs/event.go]: Btw, looks like this `ERR: ....` error message could not be shown with `ERR` to be in red right?

Comment (iamhopaul123) [reply internal/pkg/aws/cloudwatchlogs/event.go]: I wonder if we can make use of regex to do this matching stuff https://play.golang.org/p/uPTVbBXzXFz

Comment (corykitchens) [reply internal/pkg/aws/cloudwatchlogs/event.go]: Awesome thanks for all the feedback @iamhopaul123 and including a snippet of the regex!

**Review by iamhopaul123 — APPROVED** [review]
Awesome! LGTM :shipit:

**Review by efekarakus — APPROVED** [review]
awesome, thank you!

**Pull request merged**